.github/config/automation-config.json

```diff
@@ -0,0 +1,148 @@
+{
+  "area_experts": {
+    "agent_loop": {
+      "name": "Agent Loop",
+      "github_usernames": ["pgrayy", "zastrowm"]
+    },
+    "context_management": {
+      "name": "Context Management / Windows",
+      "github_usernames": ["Unshure", "afarntrog"]
+    },
+    "async_streaming": {
+      "name": "Async / Callbacks Streaming",
+      "github_usernames": ["pgrayy"]
+    },
+    "tool_executors": {
+      "name": "Tool Executors",
+      "github_usernames": ["pgrayy"]
+    },
+    "bi_directional_streaming": {
+      "name": "Bi-directional Streaming",
+      "github_usernames": ["mehtarac", "mkmeral", "pgrayy"]
+    },
+    "human_in_loop": {
+      "name": "Human in the Loop / Interrupts",
+      "github_usernames": ["mkmeral", "pgrayy"]
+    },
+    "hooks": {
+      "name": "Hooks",
+      "github_usernames": ["zastrowm"]
+    },
+    "bedrock": {
+      "name": "Bedrock Model Provider",
+      "github_usernames": ["mehtarac"]
+    },
+    "anthropic": {
+      "name": "Anthropic Model Provider",
+      "github_usernames": ["pgrayy"]
+    },
+    "gemini": {
+      "name": "Gemini Model Provider",
+      "github_usernames": ["notgitika"]
+    },
+    "litellm": {
```

Comment:
Would it be better to list the the label associated with the category?  Or are we allowing any label?

Right now all of our "area" label start with "area-"